lib/nostrum/struct/permission.ex

```diff
@@ -0,0 +1,182 @@
+defmodule Nostrum.Struct.Permission do
+  @moduledoc """
+  Functions that work on permissions.
+
+  Some functions return a list of permissions. You can use enumerable functions
+  to work with permissions:
+
+  ```Elixir
+  alias Nostrum.Cache.GuildCache
+  alias Nostrum.Struct.Guild.Member
+
+  guild = GuildCache.get!(279093381723062272)
+  member = Enum.find(guild.members, & &1.id === 177888205536886784)
+  member_perms = Member.guild_permissions(member, guild)
+
+  if :administrator in member_perms do
+    IO.puts("This user has the administrator permission.")
+  end
+  ```
+  """
+
+  use Bitwise
+
+  @type bit :: non_neg_integer
+  @type bitset :: non_neg_integer
```

Comment:
Do you think a *very* short set of docs could be useful for these? Maybe with a link to some relevant data?

Reply:
I've added in a set of docs here at 86b8184.